cpp/velox/compute/VeloxBackend.cc

```diff
@@ -332,6 +333,15 @@ arrow::Result<std::shared_ptr<ColumnarToRowConverter>> VeloxBackend::getColumnar
   }
 }
 
+std::shared_ptr<RowToColumnarConverter> VeloxBackend::getRowToColumnarConverter(
+    MemoryAllocator* allocator,
+    struct ArrowSchema* cSchema) {
+  // TODO: wait to fix task memory pool
```

Comment:
Have you create an issue for it? let's put the issue number here

Reply:
Yes, the title is issue number

Reply:
I mean the memory pool issue

Reply:
https://github.com/oap-project/gluten/issues/1108